### PR TITLE
Bump 1.10 to Marathon 1.5.11

### PR DESCRIPTION
**Pull request by alenkacz**
## High-level description

What features does this change enable? What bugs does this change fix?

MARATHON-8311 Metrics for suppress operations.
MARATHON-8304 Activate HeartbeatMonitor before Marathon is connected to Mesos.
MARATHON-8310 Ignore MARATHON_APP_ env vars.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-39258](https://jira.mesosphere.com/browse/DCOS-39258) Release Marathon 1.5.11 for DC/OS 1.10.

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): https://github.com/mesosphere/marathon/releases/tag/v1.5.11
  - [x] Test Results: https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/marathon-team-releases/job/marathon-community-release-1.5/87/console
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by alenkacz** [comment]
@mesosphere-mergebot label ready for review

**Comment by alenkacz** [comment]
@mesosphere-mergebot bump-ee

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Format of the entries must be.
 ### Notable changes
 
 * Updated REX-Ray version to 0.11.2 (DCOS_OSS-3597) [rexray v0.11.2](https://github.com/rexray/rexray/releases/tag/v0.11.2)
+* Includes Marathon [1.5.11](https://github.com/mesosphere/marathon/releases/tag/v1.5.11)
```
Comment (zen-dog) [inline CHANGES.md]: I don't think we need to put component versions into `CHANGES.md`.

**Comment by alenkacz** [comment]
@orsenthil please have a look at this weird build: https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=1125950&_focus=4014

it looks like the instance got shut down or the network shut down during the test

thanks @gpaul for looking into that with me!

**Comment by jeschkies** [comment]
@mesosphere-mergebot label Ship It

**Comment by gpaul** [comment]
@jeschkies this does not have ship it because the bump-ee PR isn't all green yet.

**Comment by gpaul** [comment]
Why that is ... I don't know. Maybe @cprovencher or @orsenthil could shed some light?

**Comment by alenkacz** [comment]
@orsenthil yes https://jira.mesosphere.com/browse/DCOS-39258

**Pull request merged**